share/jupyter/voila/templates/classic/index.html.j2

```diff
@@ -1,23 +1,32 @@
-{%- extends 'lab.tpl' -%}
+{%- extends 'nbconvert/templates/classic/index.html.j2' -%}
 
-{%- block header -%}
-<!DOCTYPE html>
-<html>
-<head>
-{%- block html_head -%}
-<meta charset="utf-8" />
-{% set nb_title = nb.metadata.get('title', '') or resources['metadata']['name'] %}
-<title>Voila: {{nb_title}}</title>
 
-<link rel="stylesheet" href="https://unpkg.com/font-awesome@4.5.0/css/font-awesome.min.css" type="text/css" />
 
 {%- block html_head_js -%}
-<script
-    src="{{resources.base_url}}voila/static/require.min.js"
-    integrity="sha256-Ae2Vz/4ePdIu6ZyI/5ZGsYnb+m0JlOmKPjt6XZ9JJkA="
-    crossorigin="anonymous">
-</script>
+{%- block html_head_js_jquery -%}
+{%- endblock html_head_js_jquery -%}
+{%- endblock html_head_js -%}
+
+
+{%- block html_head_css -%}
+  <style>
+    /*Hide empty cells*/
+    .jp-mod-noOutputs.jp-mod-noInput {
+      display: none;
+    }
+  </style>
+<link rel="stylesheet" href="https://unpkg.com/font-awesome@4.5.0/css/font-awesome.min.css" type="text/css" />
+{{ super() }}
+{%- endblock html_head_css -%}
+
+
+
+{% block custom_css %}
+{% endblock %}
+
 
+
+{% block ipywidgets %}
```

Comment:
I am not sure the block ipywidgets is the right place for notebook_execute.

I propose

 - we make an empty block `ipywidgets`
 - we make an empty block `jupyter_widgets`

to be removed when the jupyter widgets stuff becmes a macro.

The notebook execute should be its own block.

Reply:
Yes, I don't see a reason why one should be inside the other.